sqlite/src/analyze.c

```diff
@@ -2154,6 +2154,7 @@ static int loadStatTbl(
     }
     pSpace = (tRowcnt*)&pIdx->aSample[nSample];
     pIdx->aAvgEq = pSpace; pSpace += nIdxCol;
+    pIdx->pTable->tabFlags |= TF_HasStat4;
```

Comment:
I moved this code around here
https://github.com/bloomberg/comdb2/blob/823e50ba381538a00f9a02ecf81475c4d113b00a/sqlite/src/analyze.c#L2293
to make it work for 7.0